pkg/service/grpc_service_test.go

```diff
@@ -1,4 +1,4 @@
-package tests
+package service
```

Comment:
Any opinions on tests being a separate package?

On the one hand, the same package might make things easier... on the other, using a different package forces some "blackbox" testing since we lose access to the packages private identifiers.

Just thinking aloud...

Reply:
It's not just that, it also gets access to private variables